armonik/compute-plane-configmap.tf

```diff
@@ -1,24 +1,26 @@
 # configmap with all the variables
 module "compute_aggregation" {
-  source = "../utils/aggregator"
+  source   = "../utils/aggregator"
+  for_each = var.compute_plane
   conf_list = flatten([
     {
       env = {
-        ComputePlane__WorkerChannel__Address    = "/cache/armonik_worker.sock"
-        ComputePlane__WorkerChannel__SocketType = "unixdomainsocket"
-        ComputePlane__AgentChannel__Address     = "/cache/armonik_agent.sock"
-        ComputePlane__AgentChannel__SocketType  = "unixdomainsocket"
+        ComputePlane__WorkerChannel__Address    = each.value.socket_type == "tcp" ? "http://localhost:10667" : "/cache/armonik_worker.sock"
+        ComputePlane__WorkerChannel__SocketType = each.value.socket_type
+        ComputePlane__AgentChannel__Address     = each.value.socket_type == "tcp" ? "http://localhost:10666" : "/cache/armonik_agent.sock"
+        ComputePlane__AgentChannel__SocketType  = each.value.socket_type
```

Comment:
Put this into the polling_agent_aggregation and worker_aggregation from `compute-plane.tf` to avoid the for_each.